content/docs/ops/maintenance-list.md

```diff
@@ -5,48 +5,73 @@ menu:
 title: Ongoing platform maintenance
 ---
 
-The platform requires regular support and maintenance activities to remain in a compliant state. If you are on support and you can’t complete any of the items personally, you are responsible for ensuring that an appropriate person does it.
+The platform requires regular support and maintenance activities to remain in a 
+compliant state. If you are on support and you can’t complete any of the items 
+personally, you are responsible for ensuring that an appropriate person does it.
```

Comment:
Added in https://github.com/18F/cg-site/pull/922/commits/96f57d379f819e0af0564b483d9e5d0c7b18600e .